File: MPI/mpicheck/tests/functions/2.0/MPI_Add_error_string.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    int var_0;
    const char *var_1;
    int ret;
    /* calls */
    ret = MPI_Add_error_string(var_0, var_1);
    ret = PMPI_Add_error_string(var_0, var_1);
    return 0;
}

File: MPI/mpicheck/tests/functions/2.0/MPI_Add_error_string.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       CHARACTER*(10) var_1
       INTEGER var_2
        call mpi_add_error_string(var_0, var_1, var_2)
       call pmpi_add_error_string(var_0, var_1, var_2)
        end program main
    